src/libImaging/Jpeg2KEncode.c

```diff
@@ -490,6 +490,8 @@ j2k_encode_entry(Imaging im, ImagingCodecState state) {
 
     if (strcmp(im->mode, "RGBA") == 0) {
         image->comps[3].alpha = 1;
+    } else if (strcmp(im->mode, "LA") == 0) {
```

Comment:
Does it mean there was no LA support for JPEG2K at all? Do we need better testing for this case, which is not related to pdf?

Reply:
Without this change, saving an LA image as JPEG2K and visually inspecting it, shows a black rather than transparency. This fixes that. However, `getpixel()` gives `(0, 0)` either way.

When Pillow reads a JPEG2K image, it only checks the "number of components" to see if it is LA.